install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -1,14 +1,109 @@
 # Permissions and roles for istio
+# To debug: start the cluster with -vmodule=rbac,3 to enable verbose logging on RBAC DENY
+# Also helps to enable logging on apiserver 'wrap' to see the URLs.
+# Each RBAC deny needs to be mapped into a rule for the role.
+# If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
```

Comment:
If you change `ClusterRoleBinding` to simply `RoleBinding`, then you don't need to specify namespace. The binding will be scoped to whatever namespace `kubectl` is using.